### PR TITLE
Turn on markdown for druntime doc build

### PR DESCRIPTION
**Pull request by schveiguy**
I want to use the auto tester to see what happens when we turn on markdown. ~~Don't merge, I may have to fix things first.~~

### CODE REVIEW

**Comment by schveiguy** [comment]
OK I think as long as the docs come out OK, this is mergeable.

**Comment by schveiguy** [comment]
Hm... the ddox changed, how is that generated? I don't see anything in the makefile. The change is slightly worse than the original, but the original sucked too.

**Comment by schveiguy** [comment]
Even with the ddox change, I think this is good to merge.

**Comment by PetarKirov** [comment]
> Hm... the ddox changed, how is that generated? I don't see anything in the makefile.

https://github.com/dlang/dlang.org/tree/36c76465537501a0b34f32792f5a76f1f9b3888d/dpl-docs

https://github.com/dlang/dlang.org/blob/36c76465537501a0b34f32792f5a76f1f9b3888d/posix.mak#L706

**Comment by schveiguy** [comment]
@PetarKirov thanks!

Hm.. not sure I can do anything. It looks like json output doesn't spit out markdown results.

**Comment by PetarKirov** [comment]
@schveiguy Actually the `-preview=markdown` flag shouldn't affect Ddox at all, since Ddox has its own parsing [based on](https://github.com/rejectedsoftware/ddox/tree/master/source/ddox/parsers) the json files output from dmd and and libdparse. Additionally it has its own macro engine for Ddoc: https://github.com/rejectedsoftware/ddox/blob/master/source/ddox/ddoc.d

File: src/rt/dwarfeh.d [hunk 1]
```diff
@@ -746,9 +746,9 @@ LsdaResult scanLSDA(const(ubyte)* lsda, _Unwind_Ptr ip, _Unwind_Exception_Class
  *      exceptionClass = which language threw the exception
  *      lsda = pointer to LSDA table
  * Returns:
- *      >=1 means the handler index of the classType
- *      0 means classType is not in the Action Table
- *      <0 means corrupt
+ *      - &gt;=1 means the handler index of the classType
+ *      - 0 means classType is not in the Action Table
+ *      - &lt;0 means corrupt
```
Comment (PetarKirov) [inline src/rt/dwarfeh.d:751]: I wonder why was this necessary, as both Ddoc and Ddox are supposed to escape this Sunday automatically.

Comment (schveiguy) [reply src/rt/dwarfeh.d:751]: The > was interpreted as a block quote in markdown. I added the dashes to make it into a list as the glued together statements looked bad.

Comment (schveiguy) [reply src/rt/dwarfeh.d:751]: This is what it looked like before the change: http://dtest.dlang.io/artifact/website-36c76465537501a0b34f32792f5a76f1f9b3888d-01a9ae3ff87a8f9c5eb4438e29e16951/web/phobos-prerelease/rt_dwarfeh.html#.actionTableLookup

Comment (PetarKirov) [reply src/rt/dwarfeh.d:751]: Ah yes, thanks for checking
